idaes_examples/notebooks/docs/flowsheets/hda_flowsheet_with_distillation.ipynb

```diff
@@ -950,9 +976,44 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 38,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 235,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2024-08-22 10:57:12 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.H102.control_volume.properties_in[0.0].flow_mol_phase\n",
```

Comment:
Is there any way to resolve these warnings?

Reply:
I believe we are waiting for the scaling tools for this.